[2023-11-09T13:29:25.350+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=fetch_google_data_v03/run_id=scheduled__2023-11-08T00:00:00+00:00/task_id=3rd_task permission to 509
[2023-11-09T13:29:25.963+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=fetch_google_data_v03/run_id=scheduled__2023-11-08T00:00:00+00:00/task_id=3rd_task permission to 509
[2023-11-09T13:29:26.437+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_google_data_v03.3rd_task scheduled__2023-11-08T00:00:00+00:00 [queued]>
[2023-11-09T13:29:26.601+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_google_data_v03.3rd_task scheduled__2023-11-08T00:00:00+00:00 [queued]>
[2023-11-09T13:29:26.609+0000] {taskinstance.py:1361} INFO - Starting attempt 3 of 6
[2023-11-09T13:29:26.960+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): 3rd_task> on 2023-11-08 00:00:00+00:00
[2023-11-09T13:29:27.056+0000] {standard_task_runner.py:57} INFO - Started process 8580 to run task
[2023-11-09T13:29:27.065+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fetch_google_data_v03', '3rd_task', 'scheduled__2023-11-08T00:00:00+00:00', '--job-id', '481', '--raw', '--subdir', 'DAGS_FOLDER/fetch_data.py', '--cfg-path', '/tmp/tmpdx58wfaw']
[2023-11-09T13:29:27.118+0000] {standard_task_runner.py:85} INFO - Job 481: Subtask 3rd_task
[2023-11-09T13:29:27.465+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=fetch_google_data_v03/run_id=scheduled__2023-11-08T00:00:00+00:00/task_id=3rd_task permission to 509
[2023-11-09T13:29:27.540+0000] {task_command.py:416} INFO - Running <TaskInstance: fetch_google_data_v03.3rd_task scheduled__2023-11-08T00:00:00+00:00 [running]> on host a4fee2fb2c17
[2023-11-09T13:29:28.612+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Toema' AIRFLOW_CTX_DAG_ID='fetch_google_data_v03' AIRFLOW_CTX_TASK_ID='3rd_task' AIRFLOW_CTX_EXECUTION_DATE='2023-11-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-08T00:00:00+00:00'
[2023-11-09T13:29:28.626+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-09T13:29:28.672+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/src/scripts/fetch_automatic_downloads.py']
[2023-11-09T13:29:28.756+0000] {subprocess.py:86} INFO - Output:
[2023-11-09T13:30:51.487+0000] {subprocess.py:93} INFO - Downloading data for:  italy_and_regions
[2023-11-09T13:30:51.492+0000] {subprocess.py:93} INFO - Source url:  https://raw.githubusercontent.com/pcm-dpc/COVID-19/master/dati-regioni/dpc-covid19-ita-regioni.csv
[2023-11-09T13:30:51.496+0000] {subprocess.py:93} INFO - was not able to retrieve data from source_url
[2023-11-09T13:30:51.500+0000] {subprocess.py:93} INFO - Downloading data for:  nytimes
[2023-11-09T13:30:51.502+0000] {subprocess.py:93} INFO - Source url:  https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv
[2023-11-09T13:30:51.505+0000] {subprocess.py:93} INFO - was not able to retrieve data from source_url
[2023-11-09T13:30:51.508+0000] {subprocess.py:93} INFO - Downloading data for:  our_world_in_data
[2023-11-09T13:30:51.513+0000] {subprocess.py:93} INFO - Source url:  https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/owid-covid-data.csv
[2023-11-09T13:30:51.521+0000] {subprocess.py:93} INFO - was not able to retrieve data from source_url
[2023-11-09T13:30:51.526+0000] {subprocess.py:93} INFO - Downloading data for:  oxford_policy
[2023-11-09T13:30:51.530+0000] {subprocess.py:93} INFO - Source url:  https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv
[2023-11-09T13:30:51.532+0000] {subprocess.py:93} INFO - was not able to retrieve data from source_url
[2023-11-09T13:30:51.548+0000] {subprocess.py:93} INFO - Downloading data for:  spain_and_regions
[2023-11-09T13:30:51.550+0000] {subprocess.py:93} INFO - Source url:  https://cnecovid.isciii.es/covid19/resources/agregados.csv
[2023-11-09T13:30:51.553+0000] {subprocess.py:93} INFO - was not able to retrieve data from source_url
[2023-11-09T13:30:51.555+0000] {subprocess.py:93} INFO - Downloading data for:  sweden_hospitalizations
[2023-11-09T13:30:51.558+0000] {subprocess.py:93} INFO - Source url:  https://www.arcgis.com/sharing/rest/content/items/b5e7488e117749c19881cce45db13f7e/data
[2023-11-09T13:30:51.565+0000] {subprocess.py:93} INFO - was not able to retrieve data from source_url
[2023-11-09T13:30:51.567+0000] {subprocess.py:93} INFO - Downloading data for:  switzerland_regions
[2023-11-09T13:30:51.569+0000] {subprocess.py:93} INFO - Source url:  https://raw.githubusercontent.com/openZH/covid_19/master/COVID19_Fallzahlen_CH_total_v2.csv
[2023-11-09T13:30:51.572+0000] {subprocess.py:93} INFO - was not able to retrieve data from source_url
[2023-11-09T13:30:51.574+0000] {subprocess.py:93} INFO - Done with fetch_automatic_downloads.py
[2023-11-09T13:30:51.577+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-09T13:30:54.324+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fetch_google_data_v03, task_id=3rd_task, execution_date=20231108T000000, start_date=20231109T132926, end_date=20231109T133054
[2023-11-09T13:30:54.556+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-09T13:30:54.686+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
